### PR TITLE
README enhancements

### PR DESCRIPTION
**Pull request by maxidorius**
Be clearer in the intended purpose of this community

### CODE REVIEW

File: README.org
Comment (Magnap) [inline README.org]: I'm not sure what this means? We don't have the whole setup the official docs does

**Comment by Magnap** [comment]
One small comment, otherwise LGTM

**Review by Cadair — APPROVED** [review]

File: README.org
Comment (maxidorius) [reply README.org]: While we would accept other formats than the "official" documents format, it would make it harder to merge back into upstream and there would be overhead to process such documents and make them "ready".

I would like to say that, while we accept any format, we still would like contributions to follow the official format if possible.

Comment (Magnap) [reply README.org]: How about the compromise that the `matrix-doc` repo should stay as close to official spec format as possible, but `eventually-doc` can have more "freedom" to, as the README says, "facilitate this [...] occuring as quickly as possible"?

Comment (maxidorius) [reply README.org]: That's what I'm trying to say yes. How would you word it?

Comment (Magnap) [reply README.org]: That last sentence could be "However, within the `matrix-doc` repository, please follow the existing style as closely as possible, as it will make it easier to hopefully(?) merge those changes back."

Comment (maxidorius) [reply README.org]: But I would like it to apply to any repo we would fork! not just `matrix-doc`. There is only that repo currently, but a sentence that can be applied later on to repos in general would be great. Something which says: "stay as close as possible to the upstream repo format/rules/ways"

**Review by Cadair — APPROVED** [review]

File: README.org
Comment (Magnap) [reply README.org]: Sure, as long as we get to keep `eventually-doc` relatively "loose"

File: README.org
Comment (Magnap) [inline README.org]: Seems to be missing `eventually-doc` entirely?

Comment (maxidorius) [reply README.org]: I don't have a clear view of that purpose. I would add it tomorrow after a talk with @non-Jedi

File: README.org
Comment (Magnap) [reply README.org]: New proposal: "Within forked repositories, please try to match your style to upstream to ease future merging."

Comment (maxidorius) [reply README.org]: Very nice!

**Review by Magnap — APPROVED** [review]

File: README.org
Comment (non-Jedi) [inline README.org]: "Bleeding edge" could be read with negative connotations by the core team. I
think it's something that it's fine for us to aspire to in chats and stuff
or even when we're pitching the idea of Eventually Matrix, but I'd prefer we
avoid that specific language in the repo.

Maybe instead we could say that "we're experimenting with what matrix might
look like in the future." It's basically the same but less prone to someone
coming along and thinking we're trying to supercede the core team.

File: README.org
Comment (non-Jedi) [inline README.org]: Same comments as on "bleeding edge" above. Would just omit this bullet point entirely, imo.

File: README.org
Comment (non-Jedi) [inline README.org]: Should this be merged with new point introduced above?

File: README.org
Comment (non-Jedi) [inline README.org:15]: Should this be split into a separate repository from our more speculative efforts? Not sure at all.

File: README.org
Comment (non-Jedi) [reply README.org]: eventually-doc may need a rename depending on what we decide. Can see
several paths forward:

1. Maintain all docs specific to Eventually Matrix in eventually-doc
2. Split docs related to Eventually Matrix effort into a repo with efforts
   related to documenting current implementations and another repo with more
   speculative contents.
3. Just go with option 1 for now with intention to evaluate and change once
   we've tested workflow better.
   
My preference would be for 1 or 3, but I can see argument for 2.

File: README.org
Comment (non-Jedi) [inline README.org]: I had conceptualized our fork of matrix-doc as being more of a staging area
for upstreaming, but I can see the argument for making it more of a
recommended "next" implementation. Maybe omit this section from the README
until we've got a better concept of our desired workflow?

File: README.org
Comment (non-Jedi) [inline README.org]: I see this as equivalent to bullet point you added above. Merge them or choose one or the other I'd say.

File: README.org
Comment (non-Jedi) [inline README.org]: s/forked repositories/repositories forked from elsewhere/

File: README.org
Comment (non-Jedi) [inline README.org]: @Magnap one of us should go through and make sure this flows logically as an
Org document at some point. This should probably be an H2 rather than H1,
and tags should either be removed or moved to H1 heading above.

**Review by non-Jedi — COMMENTED** [review]
That's what I came up with just flying through real quick. Given the already strained relationship each of us has had at times with the core team, getting the wording right to avoid generating any more animosity is relatively important. I'll try to take a more thorough look through the README sometime soon, but overall this should be fine to merge once the references to "bleeding edge" are gone.

File: README.org
Comment (maxidorius) [reply README.org]: Since "might look like in the future" convey the message that we could be going in the wrong direction, it would also have a bad effect on people wanting to support us, as they wouldn't want to go towards something that might simply not be the good road.

"Bleeding edge" was meant to convey a message of "most current knowledge repository".
What about:
> It is the compilation of the latest and newest available information in terms of documentation, SDK and libraries.

File: README.org
Comment (maxidorius) [reply README.org:15]: We can when there is actually someone doing speculative work. So far it's only me documenting what's existing out there. I wouldn't think too far ahead here... let's create as the need arise, let's just be on the ball about it.

File: README.org
Comment (maxidorius) [reply README.org]: I'm thinking:
> Provide the latest available knowledge that the Matrix community can use to prepare for stable releases of the official specification.

File: README.org
Comment (Magnap) [reply README.org]: My vote is on 3 for the time being, but I expect that when the time comes to evaluate, I will be arguing for something very much like 2

File: README.org
Comment (maxidorius) [reply README.org]: Not in my mind as they target different use cases:
- First point is about documenting the current implementations, regardless of how good those are. It allows to build something that works with the current ecosystem.
- Second point is documenting points that the Matrix.org team agreed on and might label an existing implementation behavior as buggy. This would happen after the first point, once  you've documented something and discussed it with the Matrix team.

File: README.org
Comment (Magnap) [reply README.org:15]: I think it's very important to keep the two separate. IMO, it's a documentation vs. specification kind of deal: one documents the behavior that is seen, the other specifies the behavior that ought to exist.

File: README.org
Comment (maxidorius) [reply README.org]: Indeed, will remove this one.

File: README.org
Comment (maxidorius) [reply README.org]: > I had conceptualized our fork of matrix-doc as being more of a staging area
for upstreaming

And this is exactly the purpose, except I only listed point 1 so we can start somewhere.
To me, the overall flow would be something along the lines of:
- document what we find
- think about ourselves if it makes sense. If not talk with Matrix team to figure out if it does or not
- If it does, propose PR
- once Matrix team updated upstream with the authoritative info, we fetch and merge upstream.

I wouldn't think further until we actually got something done.

File: README.org
Comment (maxidorius) [reply README.org:15]: @Magnap do you see this blocking the README merge, as it's related to something outside this repo?

File: README.org
Comment (maxidorius) [reply README.org]: 3 for me as well. I wouldn't do more work than necessary for now. Let's see if this gains any momentum in the first place.

File: README.org
Comment (Magnap) [reply README.org:15]: If the section about what each repo is for is removed, then I don't think it does. If we keep that section in, we should do so with a clearer idea of the intended workflow. I prefer the former.

File: README.org
Comment (non-Jedi) [reply README.org]: :+1:

**Pull request merged**
